server/graphql/schema/ingestion/resolvers/mutations/ingestData.ts

```diff
@@ -289,19 +289,105 @@ class IngestDataWorker extends ResolverBase {
 
     private async validateIdentifier(identifier: IngestIdentifierInput): Promise<IdentifierResults> {
         // compute identifier; for ARKs, extract the ID from a URL that may be housing the ARK ID
+        // const vocabularyARK: DBAPI.Vocabulary | undefined = await this.getVocabularyARK();
```

Comment:
If your new code is fully tested then this 'legacy' code shouldn't be necessary. Let's clean this up. If you want to keep it around feel free to keep it outside the repo.